lib/ecto_job/job_queue.ex

```diff
@@ -175,6 +175,27 @@ defmodule EctoJob.JobQueue do
       def enqueue(multi, name, params, opts \\ []) do
         Multi.insert(multi, name, new(params, opts))
       end
+
+      @doc """
+      Requeues failed job by adding to an `Ecto.Multi` update statement,
+      which will:
+      * set `state` to `SCHEDULED`
+      * set `attempt` to `0`
+      * set `expires` to `nil`
+
+      ## Example:
+
+          Ecto.Multi.new()
+          |> MyApp.Job.requeue("requeue_job", failed_job)
+          |> MyApp.Repo.transaction()
+      """
```

Comment:
Looks good. Only suggestion is to add the error case to the typespec.

Reply:
Done.